sdk/python/scripts/test_fast.sh

```diff
@@ -4,20 +4,16 @@ PULUMI_TEST_COVERAGE_PATH=$PULUMI_TEST_COVERAGE_PATH
 
 set -euo pipefail
 
-# TODO the ignored test seems to fail in pytest but not unittest. Need
-# to trackdown why.
+mkdir -p ../../junit
+JUNIT_DIR=$(realpath ../../junit)
 
-coverage run --append -m pytest lib/test \
+coverage run --append -m pytest --junitxml "$JUNIT_DIR/python-test-fast.xml" lib/test \
              --ignore lib/test/automation \
```

Comment:
Should the trailing `\` be removed?

```suggestion
             --ignore lib/test/automation
```

Reply:
oops